src/Components/CountryCard.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+export default function CountryCard({ continent, name, iso2 }) {
+  return (
+    <>
+      <div className="country-card">
+        <div className="name">{name}</div>
+        <img src={`https://flagcdn.com/16x12/${iso2.toLowerCase()}.png`} />
+        <div className="continent">{continent}</div>
```

Comment:
Great job getting the image to display on the page with .toLowerCase(). Consider changing the color of the continent text or font-style so that the user can see the difference between the continent name and the country name :)